.github/workflows/compat-check-release-release.yml

```diff
@@ -6,72 +6,37 @@ on:
   workflow_dispatch:
 
 jobs:
-  tests_lgpu_cpp:
-    name: Lightning Compatibility test (tests_lgpu_cpp) - release/release
-    uses: ./.github/workflows/lgpu_cpp.yml
-    with:
-      lightning-version: release
-      pennylane-version: release
-    secrets: inherit
-  tests_lgpumpi_cpp:
-    name: Lightning Compatibility test (tests_lgpumpi_cpp) - release/release
-    uses: ./.github/workflows/lgpumpi_cpp.yml
-    with:
-      lightning-version: release
-      pennylane-version: release
-    secrets: inherit
-  tests_linux_cpp:
-    name: Lightning Compatibility test (tests_linux_cpp) - release/release
-    uses: ./.github/workflows/linux_cpp.yml
-    with:
-      lightning-version: release
-      pennylane-version: release
-    secrets: inherit
-  tests_lkcuda_cpp:
-    name: Lightning Compatibility test (tests_lkcuda_cpp) - release/release
-    uses: ./.github/workflows/lkcuda_cpp.yml
-    with:
-      lightning-version: release
-      pennylane-version: release
-    secrets: inherit
-  tests_windows_cpp:
-    name: Lightning Compatibility test (tests_windows_cpp) - release/release
-    uses: ./.github/workflows/windows_cpp.yml
-    with:
-      lightning-version: release
-      pennylane-version: release
-    secrets: inherit
   tests_lgpu_python:
     name: Lightning Compatibility test (tests_lgpu_python) - release/release
-    uses: ./.github/workflows/lgpu_python.yml
+    uses: ./.github/workflows/tests_lgpu_python.yml
     with:
       lightning-version: release
       pennylane-version: release
     secrets: inherit
   tests_lgpumpi_python:
     name: Lightning Compatibility test (tests_lgpumpi_python) - release/release
-    uses: ./.github/workflows/lgpumpi_python.yml
+    uses: ./.github/workflows/tests_lgpumpi_python.yml
     with:
       lightning-version: release
       pennylane-version: release
     secrets: inherit
   tests_linux_python:
     name: Lightning Compatibility test (tests_linux_python) - release/release
-    uses: ./.github/workflows/linux_python.yml
```

Comment:
Is `tests_linux_python.yml` for `lightning.qubit` or other backends?

Reply:
LQ and LK[cpu].